Cargo.toml

```diff
@@ -52,4 +53,5 @@ pyo3 = { version = "0.18", features = ["abi3-py37", "extension-module"] }
 quantum-sparse-sim = { git = "https://github.com/qir-alliance/qir-runner", rev = "e96b9a12a43e98f83f2d4d499f46d651d0c6c341", default-features = false }
 
 [profile.release]
+debug = 1
```

Comment:
Does this add value if the next line is 'strip = true'? Doesn't that strip any debug info anyway?

Reply:
I think this may have been a weird merge side effect. I removed `debug = 1` in a different change but I think these same lines were edited in the original PR this branch was made from.

Reply:
`strip=true` removes the symbols but not the debug information. There were multiple edits across the branches, so I reduced the set here so we can test the actual effect. The way it was applied before had serious performance implications.